scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -199,6 +199,34 @@ SOP-8_6.62x9.15mm_P2.54mm:
   num_pins_x: 0
   num_pins_y: 4
 
+SOP-10_3.9x4.9mm_P1mm:
+  size_source: 'https://www.holtek.com/documents/10179/11842/HT42B534-xv110.pdf'
+  body_size_x:
+    minimum: 3.80
+    maximum: 4.00
+  body_size_y:
+    minimum: 4.80
+    maximum: 5.00
+  overall_height:
+    maximum: 1.75
+
+  overall_size_x:
+    minimum: 5.90
+    maximum: 6.10
+  lead_width:
+    minimum: 0.30
+    maximum: 0.45
+  lead_len:
+    # the Holtek datasheet seems to have invalid values for this measurement
+    # (it claims a maximum of 1.27, which seems too big)
+    # so use alternate source http://www.st.com/resource/en/datasheet/viper01.pdf
+    minimum: 0.40
+    maximum: 0.90
```

Comment:
1.27 sounds plausible to me. Its the same as the `SOP-16` above. Which also matches JEDEC.